File: wms/src/main/webapp/system/app/model/acdata/acdata_OutStockDModel.js
/**
 * 模块名称：分拨出库
 * 模块编码：C201
 * 创建：hkl
 */
Ext.define('cms.model.acdata.acdata_OutStockDModel',{
	extend:'Ext.data.Model',
	idProperty:'warehouseNo, outstockNo,articleName',
	fields:[
	        {name:'warehouseNo'},
			{name:'outstockNo'},
			{name:'articleName'},
			{name:'barcodeNo'},
			{name:'inQty'},
			{name:'inWt'},
			{name:'inVl'},
			{name:'alreadyQty'},
			{name:'alreadyWt'},
			{name:'alreadyVl'},
			{name:'ostockQty'},
			{name:'ostockWt'},
			{name:'ostockVl'},
			{name:'remark'}
	       ]
});

File: model/src/main/java/com/sealinkin/acdata/model/Acdata_OutstockDModel.java
package com.sealinkin.acdata.model;

import com.sealinkin.acdata.po.Acdata_OutstockD;

public class Acdata_OutstockDModel extends Acdata_OutstockD{
	/**
	 * 
	 */
	private static final long serialVersionUID = -5023855459170267035L;
	private String warehouseNo;
	private String outstockNo;
	private String articleName;
	
	public String getArticleName() {
		return articleName;
	}
	public void setArticleName(String articleName) {
		this.articleName = articleName;
	}
	public String getWarehouseNo() {
		return warehouseNo;
	}
	public void setWarehouseNo(String warehouseNo) {
		this.warehouseNo = warehouseNo;
	}
	public String getOutstockNo() {
		return outstockNo;
	}
	public void setOutstockNo(String outstockNo) {
		this.outstockNo = outstockNo;
	}
	
}
